Packs/CommonScripts/Scripts/StixCreator/StixCreator.py

```diff
@@ -68,10 +100,58 @@ def guess_indicator_type(type_: str, val: str) -> str:
     return (auto_detect_indicator_type(val) or type_).lower()
 
 
+def create_sco_stix_uuid(xsoar_indicator: dict, stix_type: Optional[str], value: str) -> str:
+    """
+    Create uuid for sco objects.
+    """
```

Comment:
Let's extend the function description, and add the argument info. See the [google standard for python](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) (look at the `function_with_types_in_docstring` example)